R/estimate_infections.R

```diff
@@ -358,6 +368,20 @@ estimate_infections <- function(reported_cases,
     fit <- fit_model_with_vb(args,
                              verbose = verbose)
   }
+  }
+  
+  if(backend !="rstan"){
+    if (method == "exact") {
+      fit <- fit_model_with_nuts_cmd(args,
+                                 future = future,
+                                 max_execution_time = max_execution_time,
+                                 verbose = verbose)
+    }else if (method == "approximate"){
+      message("Variational Bayes Not Supported with CmdStan Backend at this Time")
```

Comment:
Proposed changes to support Variational Bayes (ADVI) algorithms are in [this](https://github.com/hsbadr/EpiNow2/commit/5dee4a52cf31e502c74bea5d32658ef96502bc9a) and [this](https://github.com/hsbadr/EpiNow2/commit/04a009649b3582b0983c76a415e93cd185463c2a) commits.

Reply:
@hsbadr cheers for filling out the rest of the args and methods! Agree completely